pkg/cgroups/cgroupcontrol.go

```diff
@@ -30,7 +30,7 @@ type Controller int
 // Group represents a control group.
 type Group string
 
-//nolint
+// nolint
```

Comment:
Why is the space removed. Machine-parsed comments should not have a space in-between, IIRC

Reply:
This change was by `gofmt`.

Reply:
Hmm, ok  nevermind then🧐